bin/run_fs_recon.py

```diff
@@ -83,48 +167,34 @@ def _run(subjects_dir, subject, raw_dir, force, mp, volume):
     if force:
         shutil.rmtree(op.join(subjects_dir, subject))
 
-    os.mkdir(op.join(subjects_dir, subject))
-    os.makedirs(op.join(subjects_dir, subject, 'mri/orig/'))
-    os.mkdir(op.join(subjects_dir, subject, 'mri/nii'))
-    fs_nii_dir = op.join(subjects_dir, subject, 'mri/nii')
+    fs_mri_dir = op.join(subjects_dir, subject, 'mri')
+    fs_nii_dir = op.join(fs_mri_dir, 'nii')
+    os.makedirs(fs_nii_dir)
+    os.mkdir(op.join(fs_mri_dir, 'orig'))
 
-    logger.info('1. Processing raw MRI data...')
-    for root, _, filenames in os.walk(parrec_dir):
+    logger.info('1. Processing T1 MRI...')
+    # trash survey scan
+    for root, _, filenames in os.walk(dicom_dir):
         for filename in fnmatch.filter(filenames, '*Quiet_Survey*'):
             os.remove(op.join(root, filename))
-    parfiles = []
-    for root, dirnames, filenames in os.walk(parrec_dir):
-        for filename in fnmatch.filter(filenames, '*.PAR'):
-            parfiles.append(op.join(root, filename))
-    parfiles.sort()
-    for pf in parfiles:
-        if (volume in pf) or ('FLASH' in pf):
-            print('Converting {0}'.format(pf))
-            pimg = nibabel.load(pf)
-            pr_hdr = pimg.header
-            raw_data = pimg.dataobj.get_unscaled()
-            affine = pr_hdr.get_affine(origin='fov')
-            nimg = nibabel.Nifti1Image(raw_data, affine, pr_hdr)
-            nimg.to_filename(op.join(parrec_dir, op.basename(pf)[:-4]))
-            shutil.copy(nimg.get_filename(), fs_nii_dir)
-
-    for ff in glob.glob(op.join(fs_nii_dir, '*.nii')):
-        if volume in op.basename(ff):
-            os.symlink(ff, op.join(fs_nii_dir, 'MPRAGE.nii'))
-        elif 'FLASH5' in op.basename(ff):
-            os.symlink(ff, op.join(fs_nii_dir, 'FLASH5.nii'))
-        elif 'FLASH30' in op.basename(ff):
-            os.symlink(ff, op.join(fs_nii_dir, 'FLASH30.nii'))
-
-    logger.info('2. Starting FreeSurfer reconstruction process...')
-    mri = op.join(fs_nii_dir, 'MPRAGE.nii')
-    run_subprocess(['mri_concat', '--rms', '--i', mri,
-                    '--o', op.join(subjects_dir, subject, 'mri/orig/001.mgz')],
-                   env=this_env)
-    run_subprocess(['recon-all', '-openmp', '%.0f' % mp, '-subject', subject, '-all'], env=this_env)
-    for morph_to in ['fsaverage', subject]:
-        run_subprocess(['mne_make_morph_maps', '--to', morph_to, '--from', subject], env=this_env)
-
+    t1_parrec = find_files(dicom_dir, '*%s*.PAR' % volume)[0]
+    t1_nii = op.join(fs_nii_dir, op.basename(t1_parrec[:-4] + '.nii'))
+    parrec_to_nii(t1_parrec, fs_nii_dir)
+    logger.info('2. Starting FreeSurfer reconstruction...')
+    run_subprocess(['mri_concat', '--rms', '--i', t1_nii,
+                    '--o', op.join(subjects_dir, subject,
+                                   'mri/orig/001.mgz')], env=this_env)
```

Comment:
At some point we should look into doing this with `nibabel`. I suspect this just takes the RMS of the data (across echoes) and writes the result to disk. If you're feeling bored it might be fun to check.